File: 22-03-18/python/01-Monk-Takes-a-walk.py
t = int(input())
for _ in range(0,t):
    data = input().lower()
    cnt = 0
    for i in data:
        if(i == "a" or
           i == "e" or
           i == "i" or
           i == "o" or
           i == "u" 
           ):
               cnt += 1
    print(cnt)
    

File: Mar-18/22-03-18/java/01-Monk-Takes-a-walk.java

import java.util.*;
import java.io.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        while(t-- > 0)
        {
            String str = input.next().toLowerCase();
            int len = str.length();
            int cnt = 0;
            
            for(int i = 0; i < len; i++)
            {
                if(str.charAt(i) == 'a' ||
                   str.charAt(i) == 'e' ||
                   str.charAt(i) == 'i' ||
                   str.charAt(i) == 'o' ||
                   str.charAt(i) == 'u' 
                   )
                   cnt++;
            }
            System.out.println(cnt);
        }
        
    }
}
